src/main/java/net/pms/update/AutoUpdaterServerProperties.java

```diff
@@ -46,6 +46,12 @@ private String getStringWithDefault(String key, String defaultValue) {
 	}
 
 	private String getPlatformSpecificKey(String key) {
-		return key + "." + operatingSystem.toString();
+		String os = operatingSystem.toString();
+		 //we use only the one UMS Windows version so the exact OS Windows version is not relevant
+		if (os.startsWith("windows")){
+			os = "windows";
+		}
```

Comment:
This should make it work on macOS too

```suggestion
		if (!os.startsWith("linux")) {
			os = operatingSystem.getPlatformName();
		}
```